### DISCUSSION TITLE
Add Zoho Calendar

### DISCUSSION THREAD

**Discussion by lrpradeepkumar-zoho** [add2cal/add-to-calendar-button#352] [Ideas / Requests]
Add Zoho calendar to the list

**Comment by jekuer** [comment]
_First update._
Currently not (or no longer) possible.
We are in contact with ZOHO about it. 🤞🏼

**Comment by jekuer** [comment]
_Next update:_
It is currently not possible at all.
Discussed with ZOHO and they have it on their roadmap. Requirements from our side got added.
For now, I would convert this into a discussion, since there is nothing we can do as long as they did not play their part.
